src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseMultiSignMessage.cs

```diff
@@ -328,12 +594,33 @@ public override int GetEncodedLength()
             return encodedLength;
         }
 
+        /// <summary>
+        /// Attempts to encode this message into the specified buffer.
+        /// </summary>
+        /// <param name="destination">The buffer in which to write the encoded value.</param>
+        /// <param name="bytesWritten">On success, receives the number of bytes written to <paramref name="destination" />.</param>
+        /// <returns><see langword="true" /> if <paramref name="destination" /> had sufficient length to receive the value; otherwise, <see langword="false" />.</returns>
+        /// <remarks>Use <see cref="GetEncodedLength()"/> to determine how many bytes result in encoding this message.</remarks>
+        /// <exception cref="InvalidOperationException">
+        ///   <para>
+        ///     The <see cref="CoseMessage.ProtectedHeaders" /> and <see cref="CoseMessage.UnprotectedHeaders" /> collections have one or more labels in common.
+        ///   </para>
+        ///   <para>-or-</para>
+        ///   <para>
+        ///     Message must carry at least one signature.
```

Comment:
The rule for the sentence fragments in exceptions is to word them as if they were preceded them with "Throws {type} when ".

"Throws InvalidOperationException when message must carry at least one signature."  doesn't make sense to me.  I'm also not quite sure what it means to help reword it.

Reply:
Ah, looking at the code, I think this is something like

```suggestion
        ///     All signatures have been removed from the <see cref="Signatures" /> collection.
```

Reply:
> word them as if they were preceded them with "Throws {type} when ".

In that case, this should be:
```suggestion
        ///     Message does not contain at least one signature.
```

Reply:
> All signatures have been removed from the <see cref="Signatures" /> collection.

That's another way of saying it.

Reply:
Since Encode won't produce a message with no signatures, I assume that Decode won't decode one; so "you removed it" seems like the only way that it hits zero.  On that basis, I like the "removed" version better.